### DISCUSSION TITLE
Enhance the What's New / Announcements HUD with a category name and icon

### DISCUSSION THREAD

**Discussion by khalwat** [craftcms/cms#9747] [Ideas] [answered]
Suggestion: maybe just like Apple's Notifications, the icon of the thing registering the new item should be displayed alongside the title of the notification?

That'd make it clear whether the item was coming from Craft, or Commerce, or some other plugin.

e.g.:
<img width="363" alt="Screen Shot 2021-06-14 at 11 32 53 PM" src="https://user-images.githubusercontent.com/7570798/121988767-ebb91400-cd68-11eb-8a21-6a47f5a25c65.png">

Here's what it looks like now for reference:

<img width="414" alt="Screen_Shot_2021-08-17_at_10 19 13_PM" src="https://user-images.githubusercontent.com/7570798/130262376-94f7cbfe-e043-4286-b0cf-f15f026c4297.png">

**Comment by brandonkelly** [comment, marked as answer]
Made this change for Craft 4 (#9748).

<img width="424" alt="The “What’s New” HUD, listing two previously-read announcements from Craft CMS, and one unread announcement from SEOmatic." src="https://user-images.githubusercontent.com/47792/130328980-f627150a-01b8-4e35-a7c3-8a1048fc744c.png">

**Reply by khalwat** [reply]
Looks amazing, thank you!
